app/components/user-search.cjsx

```diff
@@ -12,7 +12,7 @@ module.exports = React.createClass
       apiClient.type('users').get display_name: value, page_size: 10
         .then (users) =>
           opts = for user in users
-            { value: user.id, label: user.display_name }
+            { value: user.id, label: "@#{ user.login }: #{ user.display_name }" }
```

Comment:
Might make sense to only include the display name or reverse the order it's being shown since it's searching by display_name.